test/test_scenario_1.py

```diff
@@ -12,21 +12,23 @@
 
 SCENARIO_DIR = os.path.join(TEST_DIR, "scenario_1")
 
-
-@pytest.mark.skip("Test fails, needs to be fixed")
 def test_tropo_delay(tmp_path):
+    '''
+    Scenario: 
+    1: Small area, ERA5, Zenith delay
+    '''
     lats = gdal_open(os.path.join(
-        SCENARIO_DIR, 'geom', 'ERA5_Lat_2018_01_01_T00_00_00.dat'
+        SCENARIO_DIR, 'geom', 'lat.dat'
     ))
     lons = gdal_open(os.path.join(
-        SCENARIO_DIR, 'geom', 'ERA5_Lon_2018_01_01_T00_00_00.dat'
+        SCENARIO_DIR, 'geom', 'lon.dat'
     ))
 
     time = datetime.datetime(2020, 1, 3, 23, 0)
 
     _, model_obj = modelName2Module("ERA5")
     wet_file, hydro_file = makeDelayFileNames(
-        time, Zenith, "envi", "ERA5", tmp_path
+        time, Zenith, "envi", "ERA5", SCENARIO_DIR
```

Comment:
Test should be isolated to a temp directory to avoid interactions with previous runs.